tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,24 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+        org = make_org(cached=True)
+        cv = make_content_view({u'organization-id': org['id']})
+        product_info = make_product({u'organization-id': org['id']})
+        repo = self._make_file_repository_upload_contents(
+            {u'content-type': 'file', u'organization-id': org['id'], u'product-id':
+                product_info['id']})
+        ContentView.add_repository({u'id': cv['id'], u'repository-id': repo['id'],
+                                    u'organization-id': org["name"]})
+        env = make_lifecycle_environment({u'organization-id': org['id']})
+        ContentView.publish({u'id': cv['id']})
+        content_view_info = ContentView.version_info({ u'content-view-id': cv['id'],
+                                                       u'content-view': cv['name'], u'version': 1})
```

Comment:
I think you need not to pass both id and name of cv to get the info, either one is enough.

Reply:
fixed.